modules/fedimint-dummy-client/src/db.rs

```diff
@@ -17,6 +17,12 @@ pub enum DbKeyPrefix {
     // Used to verify that 0x50 key can be written to, which used to conflict with
     // `DatabaseVersionKeyV0`
     ClientName = 0x50,
+    /// Prefixes between 0xb1..=0xcf shall all be considered allocated for
```

Comment:
In global range we were kind of retro fitting it, where `0xb0` was already known to be used for "user data", `0xff` already used for module prefix.

We probably want to reserve `0xb0..0xd0` for user data (external, applications integrating / forking modules), and possibly `0xd0..=0xff` for "fedimint core's internal use". I don't think the core use will be ever necessary, but it doesn't really hurt, and maybe just maybe at some point we are going to need it. In a pinch, it could eventually it could be reallocated to "more user data".

Reply:
updated